pulsetranslator/loghandler.py

```diff
@@ -128,6 +128,7 @@ def publish_build_message(self, data):
         for tag in data['tags']:
             if tag:
                 key_parts.append(tag)
+        key_parts.append(data['locale'])
```

Comment:
That would be a no-op given that we always have a locale present, which will be `en-US`. What we might be able to do is to only add the locale if the `l10n` tag is present. What do you think about it?